coordinator/src/main/scala/filodb.coordinator/IngestionActor.scala

```diff
@@ -265,9 +265,10 @@ private[filodb] final class IngestionActor(dataset: Dataset,
           off }
         .until(_ >= endOffset)
         // TODO: move this code to TimeSeriesShard itself.  Shard should control the thread
-        .lastL.runAsync(shardInstance.ingestSched)
+        .lastOptionL.runAsync(shardInstance.ingestSched)
```

Comment:
When would this ever return None?   Not too clear on semantics of this

Reply:
See the other changes in commit. None would be returned when endOffset < startOffset.
This is required to eagerly move state from Recovering to Active if there is no data in kafka.